store/todos.js

```diff
@@ -124,4 +139,33 @@ export const mutations = {
     state.perPage = 9;
     state.page = 1;
   },
+  setListPerPage(state, val) {
+    state.taskListPerPage = state.taskList;
+  },
+  filterTaskList(state, val) {
+    state.filterOptions.map((option) => (option.status = false));
```

Comment:
What are we mapping this into?

Reply:
mapping option.status to false in every element

Reply:
We can use forEach if we want change an array inline. Map returns a new mapped array which we aren't using here.

Reply:
Resolved